src/cross-browser-ui/static/_locales/cs/messages.json

```diff
@@ -0,0 +1,74 @@
+{
+  "extensionName": {
+    "message": "Bergamot",
```

Comment:
The English original is "Bergamot Browser Extension", which has no ideal translation to Czech mostly due to noun compounding. Its literal translation would be "Rozšíření prohlížeče Bergamot", which is problematic for two reasons:

1. unlike in English, the most important word "Bergamot" is in the last position, making it visually less important
2. an alternative parsing of the compound suggests that "Bergamot" is the name of the browser :)

Because of that we agree to keep the name simple, just "Bergamot". It should be clear from the context that it is in fact a browser extension.